### PR TITLE
Normalista Styleguide v0

### PR DESCRIPTION
**Pull request by davo**
@miguelpaz @jueyang this is the version I've been working on the last couple of days.

Please check the stack, I update the build document (see here https://github.com/davo/normalista/blob/master/BUILD.md).

screen shot 2016-10-30 at 8 51 43 pm

### CODE REVIEW

**Comment by miguelpaz** [comment]
Thanks. I will check everything and wait for @jueyang's comments too.

**Comment by miguelpaz** [comment]
@davo and @jueyang I get this: 

> This branch has conflicts that must be resolved. Conflicting files
> BUILD.md
> _includes/head.html

**Comment by miguelpaz** [comment]
Just saw the modifications in BUILD.md and in _includes/head.html. Before accepting the pull request, please @jueyang can you check that changes in _includes/head.html won´t break anything, and how to merge both build.md to not loose the existing documentation but rather update it?

**Comment by jueyang** [comment]
Hi @davo. Great new flow. I have a few suggestions.
1. I ran into missing gem warnings using the current flow. Could you add the installation of `bundler` and run `bundle install` as part of the installation in `BUILD.md`. I'm not sure what order you'd like to put it. 
2. Style guide is awesome! I guess the next step would be to add appropriate styles to other layouts. Do you think those happen in the next PR?

I did a small modification of `BUILD.md` and `package.json` to consolidate `gulp` installation based on your PR. See https://github.com/jueyang/normalista/tree/davo-master

File: BUILD.md [hunk 1]
```diff
@@ -1,12 +1,24 @@
+#### Normalista uses [Github Pages](https://help.github.com/articles/using-jekyll-as-a-static-site-generator-with-github-pages/) to make easy to mantain and publish academic content.
+
+The stack is based on gulp and jekyll.
+
 ## Installation
 
-Install jekyll following http://jekyllrb.com/docs/installation/
+Install [jekyll](http://jekyllrb.com/docs/installation/).
+
+Install [npm](https://docs.npmjs.com/getting-started/installing-node) via node.
```
Comment (impronunciable) [inline BUILD.md]: So the requirement would be to have node.js installed. Node ships with npm. Most people know if they have node, but not if they have npm. Saying that "node.js" is the requirement will make more sense. Also it's easier to just go and install node in any platform

File: BUILD.md [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+
+Install [gulp](https://github.com/gulpjs/gulp/blob/master/docs/getting-started.md).
```
Comment (impronunciable) [inline BUILD.md]: You can add gulp as a devDependency, removing this extra prerequisite

File: BUILD.md [hunk 3]
```diff
@@ -1,12 +1,24 @@
...
+
+Run `npm install`.
+
+Run `gulp`.
```
Comment (impronunciable) [inline BUILD.md:13]: If you add gulp in your devDependencies you can have a npm script, say `run` as :`"run": "gulp"`

File: _includes/footer.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<footer>
+	<div class="back-to-top tr pv5 pr5 back"><a href="" class="no-underline">Back to the top</a></div>
```
Comment (impronunciable) [inline _includes/footer.html:2]: I could be wrong but an empty href means no cursor pointer. Using `href="#"` adds it

File: _sass/scss/_lists.scss [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+
+/*
+
+   LISTS
+   http://tachyons.io/docs/elements/lists/
+
+*/
+
+.list {         list-style-type: none; }
```
Comment (impronunciable) [inline _sass/scss/_lists.scss:15]: Is this indentation intended?

**Comment by miguelpaz** [comment]
Amazing job

File: _sass/scss/_lists.scss (hunk 5 above)
Comment (davo) [reply _sass/scss/_lists.scss:15]: That's part of Tachyons, I didn't see it :/

**Comment by miguelpaz** [comment]
Should I close @davo ?

**Pull request merged**